.devops/deploy-pipelines.yml

```
@@ -329,3 +329,72 @@ stages:
                     VALUE_FILE: "helm/pay-wallet-values-uat.yaml"
                     GREEN_VERSION: $(app_version)
   # --- END Deploy UAT --- #
+
+  # --- START of Validate event in Hub (UAT) --- #
+
+  - stage: "UAT_Testing_Generate_Wallet_Event"
+    displayName: "UAT Testing Generate Wallet Event"
+    dependsOn: [ Deploy_UAT_Green]
+    pool:
+      vmImage: 'ubuntu-latest'
+    jobs:
+      - job: ValidateCDCFlow
+        displayName: 'Run API Tests DataLake Flow (UAT)'
+        steps:
+          - script: |
+              yarn global add newman
+            displayName: 'newman installation'
+
```

```suggestion
          - script: |
              yarn global add newman
            displayName: 'newman installation'
          - checkout: pagopaWalletTests
```
Perform checkout to `pagopa-wallet-tests` repo here before launching newman test execution.
This way current folder will be set to the repository root folder and tests could be run pointing to remote repository files instead of duplicating collections into this repo

---

Unresolved conversation. Please perform checkout on the wallet tests here. Do not duplicate test collections since any modification would need to be done in n repositories. the main purpose of having a centralized test repo is to reuse collections